drivers/scheduler/k8s/specs/elasticsearch-crd-webhook/pure/pure-storage-class.yaml

```diff
@@ -0,0 +1,17 @@
+kind: StorageClass
```

Comment:
Why is this storage class for direct flash needed here?

Reply:
It's present in all of the other specs, so I kept it.

Reply:
It's not needed from px-backup end please remove it.